packages/button/src/Default/styles.module.scss

```diff
@@ -18,7 +19,7 @@
       --bs-btn-active-color: #{theme.$owl-ref-palette-white};
       --bs-btn-active-bg: var(--bs-gray-800);
       --bs-btn-active-border-color: var(--bs-gray-800);
-      --bs-btn-active-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+      --bs-btn-active-shadow: inset 0 3px 5px rgb(0 0 0 / 12.5%);
```

Comment:
Looking gooood. Not sure if this change was supposed to be in this PR though.

Other than that, this is good to go! 🍻